### ISSUE TITLE
DataLossError 

### ISSUE THREAD

**Issue by rishimadhok** [KranthiGV/Pretrained-Show-and-Tell-model#17]
Hi, thank you providing the pre trained model. However, while running the last instruction, i.e 
**Run inference to generate captions.**

bazel-bin/im2txt/run_inference \
  --checkpoint_path=${CHECKPOINT_PATH} \
  --vocab_file=${VOCAB_FILE} \
  --input_files=${IMAGE_FILE}

I am getting the following error:

**DataLossError (see above for traceback): Unable to open table file /Users/rishimadhok/Desktop/Quote_my_picture/im2txt/im2txt/data/Pretrained-Show-and-Tell-model/model.ckpt-1000000.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?
	 [[Node: save/RestoreV2_382 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_arg_save/Const_0_0, save/RestoreV2_382/tensor_names, save/RestoreV2_382/shape_and_slices)]]**

Please let me know what should I do to resolve this error? Thank you for your time!

**Comment by ivishnevs** [comment]
Getting same error; @rishimadhok do you resolve this issue?

**Comment by zhmd** [comment]
How do you set the ${CHECKPOINT_PATH}? I have the same error as you and it turns out that the variable should be set to something like:
CHECKPOINT_PATH="${path_to_where_you_put_it}/new_model.ckpt-2000000"

P.S. Since I am using a relatively new tensorflow version, I have to convert the model as described [here](https://github.com/KranthiGV/Pretrained-Show-and-Tell-model/issues/9)
